### DISCUSSION TITLE
[A feature Request] Starbound-esque recoloring system

### DISCUSSION THREAD

**Discussion by triplezeta** [shiptest-ss13/Shiptest#157] [Ideasguy]
Starmound has a dye system which, instead of just recoloring something multiplicatively, will replace pixels of one color with another for the clothing item its applied to. 

I've wanted something like this in ss13 for quite some time now. Such a system would be very nice to have here and allow for huge flexibility for player outfits.

**Comment by rye-rice** [comment]
i believe cit's polychrome system fits that bill? i think a port was attempted in the previous codebase but i don't remember why it was closed

**Reply by triplezeta** [reply]
iirc polychrome just recolors stuff with multiplication.

**Reply by triplezeta** [reply]
or it uses the color matrix stuff. iunno
